packages/hdwallet-trezor/src/trezor.ts

```diff
@@ -47,7 +47,7 @@ import * as Btc from './bitcoin'
 import * as Eth from './ethereum'
 import { TrezorTransport } from './transport'
 
-export function isTrezor(wallet: HDWallet): wallet is TrezorHDWallet {
+export function isTrezor(wallet: any): wallet is TrezorHDWallet {
```

Comment:
likewise

Reply:
and the `isKeepKey` one should be made to take an `HDWallet` instead of an `any`